java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -22,9 +22,15 @@ public interface ServerManagementClusterCommands {
      * @return Response from Redis cluster with a <code>Map{@literal <String, String>}</code> with
      *     each address as the key and its corresponding value is the information for the node.
      * @example
-     *     <p><code>
-     *     {@literal Map<String, String>} routedInfoResult = clusterClient.info().get().getMultiValue();
-     *     </code>
+     *     <pre>{@code
+     * ClusterValue<String> payload = clusterClient.info().get();
+     * // By default, the command is routed to multiple nodes, expecting a MultiValue result.
+     * for (Map.Entry<String, String> entry : payload.getMultiValue().entrySet()) {
+     *     String nodeName = entry.getKey();
```

Comment:
minor: nothing is done with this variable.

Reply:
maybe:
```java
Map<String, String> infoResult = clusterClient.info().get().getMultiValue();
for (String data : infoResult) {
    assert data.contains("cluster_enabled:1");
}
```

Reply:
you can print it, if you want.